modules/nw-bpfman-operator-installing-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/bpfman-operator/bpfman-operator-install.adoc
+
+////
+Operator Hub capitalizes all Operator names; officially, it is bpfman though.
+////
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-bpfman-operator-installing-console_{context}"]
+= Installing the bpfman Operator using the web console
+
+As a cluster administrator, you can install the bpfman Operator using the web console.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have an account with administrator privileges.
+
+.Procedure
+
+. Install the bpfman Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
```

Comment:
This renders as "{product-title}".  I'm not sure if that's supposed to be replaced by the actual product title.

Reply:
@anfredette, it does on GitHub, yes. It shouldn't in the published documentation.